doc/sphinx-guides/source/developers/dev-environment.rst

```diff
@@ -171,14 +183,181 @@ For a list of possible values, please "find usages" on the settings key above an
 
 Now when you go to http://localhost:8080/oauth2/firstLogin.xhtml you should be prompted to create a Shibboleth account.
 
-Rebuilding your dev environment
--------------------------------
+Geoconnect
+----------
 
-If you have an old copy of the database and old Solr data and want to start fresh, here are the recommended steps: 
+Set Up
+~~~~~~
 
-- drop your old database
-- clear out your existing Solr index: ``scripts/search/clear``
-- run the installer script above - it will create the db, deploy the app, populate the db with reference data and run all the scripts that create the domain metadata fields. You no longer need to perform these steps separately.
-- confirm you are using the latest Dataverse-specific Solr schema.xml per the "Installing and Running Solr" section of this guide
-- confirm http://localhost:8080 is up
-- If you want to set some dataset-specific facets, go to the root dataverse (or any dataverse; the selections can be inherited) and click "General Information" and make choices under "Select Facets". There is a ticket to automate this: https://github.com/IQSS/dataverse/issues/619
+Geoconnect works as a middle layer, allowing geospatial data files in Dataverse to be visualized with Harvard WorldMap. To set up a Geoconnect development environment, you can follow the steps outlined in the `local_setup.md <https://github.com/IQSS/geoconnect/blob/master/local_setup.md>`_ guide. You will need Python and a few other prerequisites.
+
+How Dataverse Ingests Shapefiles
```

Comment:
All of this detail here and below about Geoconnect is great but I'm thinking we should have a separate page for it. This page is getting long and the focus should be about setting up a development environment, not a relatively deep dive into a specific feature. I would suggest following the pattern established by UNF, which is even split into multiple pages:

- http://guides.dataverse.org/en/4.6.1/developers/unf/index.html
  - http://guides.dataverse.org/en/4.6.1/developers/unf/unf-v3.html
  - http://guides.dataverse.org/en/4.6.1/developers/unf/unf-v5.html
  - http://guides.dataverse.org/en/4.6.1/developers/unf/unf-v6.html

I don't think we want all that detail about UNF on this "Development Environment" page.